doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -171,4 +209,83 @@ For each empty box in the matrix above, ensure that the following applications c
     sudo snap run mir-test-tools.mir-flutter-app
    
     echo All tests passed
-   ```
\ No newline at end of file
+   ```
+
+## Display Configuration
+- Validate that changing the scale of an output via the display configuration
+  works properly. This was first encounted in [#3165](https://github.com/canonical/mir/issues/3165).
+  To test:
+    1. Use `gbm-kms` as the display platform and start a Mir compositor with the `--display-config static=$FILENAME` flag
+    2. Open up `$FILENAME` and change the scale of an output. Save the file.
+    3. Confirm that the scene on that output is _not_ misrendered
+
+- Validate that different orientations in the display config result in the
+  correct rotation being applied to the final image. This issue was first
+  encountered in [this issue](https://github.com/canonical/mir/issues/3185). To test:
+    1. Start a Mir compositor with the `--display-config static=$FILENAME` flag
+    2. Set orientation of your outputs in `$FILENAME` [normal, left, right, inverted]
+    3. Confirm that the output has the correctly rotated content
+
+- Validate that custom attrbutes attributes can be provided in the display configuration. This
+  was first encountered in [#172](https://github.com/canonical/ubuntu-frame/issues/172). To
+  test:
+  1. Start a Mir compositor with the `--display-config static=$FILENAME` flag
+  2. In the configuration, have a custom attribute, e.g. `snap-name` in the following example:
```

Comment:
The only compositor that supports `snap-name` is Frame. And that gets its display configuration from the `display` snap configuration option.